corehq/apps/domain/migrations/0008_geocoder.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.privileges import GEOCODER
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        GEOCODER,
+        skip_edition='Paused,Community,Standard,Pro,Advanced',
+        noinput=True,
+    )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('domain', '0007_auto_20200924_1753'),
```

Comment:
Can you move this into accounting, for the sake of consistency? All other migrations that call `cchq_prbac_grandfather_privs` are in accounting.